### ISSUE TITLE
Craft commands erroring on startup

### ISSUE THREAD

**Issue by bencroker** [nystudio107/spin-up-craft#2]
It appears that Craft commands are being executed _before_ the DB is ready on running `make dev`, resulting in some errors.

Is there anyway that those commands can be run only once seeding the DB has completed and the DB is ready to handle requests?

<img width="1144" alt="Screenshot 2023-04-25 at 09 32 32" src="https://user-images.githubusercontent.com/57572400/234329210-fb7f71cb-e290-4479-82f9-672bb40ce898.png">

**Comment by khalwat** [comment]
Can you give me a steps to reproduce here?

**Comment by khalwat** [comment]
I just did:

```
make nuke
```

...from the bare `spin-up-craft` repo, which should flatline everything, and restart from scratch, and it worked without incident:

```
spin-up-craft-php-1    | Use the `composer fund` command to find out more!
spin-up-craft-php-1    | > post-update-cmd: yii\composer\Plugin->showUpgradeNotes
spin-up-craft-php-1    | [25-Apr-2023 16:18:43] NOTICE: fpm is running, pid 6
spin-up-craft-php-1    | [25-Apr-2023 16:18:43] NOTICE: ready to handle connections
spin-up-craft-queue-1  | Checking for pending Craft and plugin migrations ...
spin-up-craft-queue-1  | No new migrations found. Your system is up to date.
spin-up-craft-queue-1  | 
spin-up-craft-queue-1  | Checking for pending content migrations ...
spin-up-craft-queue-1  | No new migrations found. Your system is up-to-date.
spin-up-craft-queue-1  | 
spin-up-craft-queue-1  | ### Your Craft site is ready!
spin-up-craft-queue-1  | Frontend URL: http://localhost:8050/
spin-up-craft-queue-1  | CP URL: http://localhost:8050/admin
spin-up-craft-queue-1  | CP User: admin
spin-up-craft-queue-1  | CP Password: project
```

**Comment by bencroker** [comment]
Sorry I wasn’t more specific. This happens when opening in a codespace.

**Comment by khalwat** [comment]
So this was fun.

Couldn't reproduce it locally, but apparently some kind of latency involved with Codespaces would cause a situation where the database would respond, but it had not been fully populated with data yet.

This only affects the initial spin up of the container, when the db is initially seeded.

Addressed in: https://github.com/nystudio107/spin-up-craft/commit/1620a79e9b02f0e2fe289279bac67fcc80282859

Please verify the fix @bencroker -- ensure that when you start a Codespace, you do so from the `develop` branch, via the **Code** button and **Codespaces** tab, where you can set more specific options.

**Comment by bencroker** [comment]
Tested the `develop` branch and your new checks work a treat, thanks!

<img width="922" alt="Screenshot 2023-04-25 at 20 45 58" src="https://user-images.githubusercontent.com/57572400/234444875-ef1f855e-c2ba-4cbc-b710-1068b6a4ccd6.png">

**Comment by khalwat** [comment]
Thx, will cut a release.

**Comment by khalwat** [comment]
Released as 1.0.14 -> https://github.com/nystudio107/spin-up-craft/releases/tag/1.0.14
